### PR TITLE
Add cli command execution for Evaluate initialize and start_trial

### LINKED ISSUE #48: Add CLI execution for evaluator
For easy-to-use, add CLI execution.

### PR DESCRIPTION
**Pull request by vkehfdl1**
close #48

- Add cli command `autorag evaluate`
- load version from VERSION file.
- add evaluate_generation to evaluate/__init__.py
- edit README.md to latest usage.

### CODE REVIEW

**Review by bwook00 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -45,35 +45,37 @@ evaluate_dataset.to_parquet('your/path/to/evaluate_dataset.parquet')
...
+from autorag.evaluator import Evaluator
 
-evaluator = Evaluator(qa_path='your/path/to/qa.parquet', corpus_path='your/path/to/corpus.parquet')
+evaluator = Evaluator(qa_data_path='your/path/to/qa.parquet', corpus_data_path='your/path/to/corpus.parquet')
 evaluator.start_trial('your/path/to/config.yaml')
 ```
 or you can use command line interface
 ```bash
-autorag evaluate --config your/path/to/default_config.yaml
+autorag evaluate --config your/path/to/default_config.yaml --qa_path your/path/to/qa.parquet --corpus_path your/path/to/corpus.parquet
```
Comment (Eastsidegunn) [inline README.md:55]: how about unify variable name of xxx_data_path and xxx_path
+ in this version, qa_path variable called by cli variable?

**Review by Eastsidegunn — DISMISSED** [review]
LGTM

**Review by Eastsidegunn — APPROVED** [review]
LGTM

**Review by bwook00 — APPROVED** [review]
LGTM

**Pull request merged**